apps/platformServices/spiService.adef

```diff
@@ -31,7 +31,23 @@ requires:
 {
     device:
     {
+#if ${LEGATO_TARGET} = wp85
         [rw] /dev/spidev0.0 /dev/
+#elif ${LEGATO_TARGET} = wp750x
+        [rw] /dev/spidev0.0 /dev/
+#elif ${LEGATO_TARGET} = wp76xx
+        [rw] /dev/spidev1.0 /dev/
+#elif ${LEGATO_TARGET} = wp77xx
+        [rw] /dev/spidev1.0 /dev/
+#elif ${LEGATO_TARGET} = ar7
+        // TODO: ar7 device path
+#elif ${LEGATO_TARGET} = ar758x
+        // TODO: ar758x device path
+#elif ${LEGATO_TARGET} = ar86
+        // TODO: ar86 device path
+#elif ${LEGATO_TARGET} = em75xx
+        // TODO: em75xx device path
```

Comment:
I would think that /dev/spidev0.0 should be a setting in targets/ with a value that defaults to /dev/spidev0.0

Reply:
Do you mean something like this?

*target/mdm9x07.sinc*
```
buildVars:
{
...
    PRIMARY_SPI_DEVICE = '/dev/spidev1.0'
...
}
```

*target/mdm9x15.sinc*
```
buildVars:
{
...
    PRIMARY_SPI_DEVICE = '/dev/spidev0.0'
...
}
```

Reply:
More like `SPI_PRIMARY_DEV = /dev/spidev1.0` in mdm9x07.sinc and something like
```
buildVars:
{
  #if ${SPI_PRIMARY_DEV} = ""
     SPI_PRIMARY_DEV_CHOICE = "/dev/spidev0.0"
  #else
     SPI_PRIMARY_DEV_CHOICE = ${SPI_PRIMARY_DEV}
  #endif
}

requires:
{
    device:
    {
        [rw] ${SPI_PRIMARY_DEV_CHOICE}
    }
}
```

Logic is a bit complex because a variable that has been used cannot be redefined in mktools, but eventually this provides a way to the device and a reasonable default.